src/fluid.h

```diff
@@ -29,7 +29,7 @@ class CFluidParameters {
     static const int FLUID_ACTIVATE_HEIGHT = 10;
     static const int64_t MAX_FLUID_TIME_DISTORT = 60 * 60; // Maximum time distort = 1 hour.
     static const CAmount FLUID_TRANSACTION_COST = 100000 * COIN; // Cost to send a fluid transaction
-    static const CAmount FLUID_MAX_REWARD_FOR_DYNODE_ = 1000 * COIN; // Max dynode block reward using fluid OP_REWARD_DYNODE
+    static const CAmount FLUID_MAX_REWARD_FOR_DYNODE = 1000 * COIN; // Max dynode block reward using fluid OP_REWARD_DYNODE
     static const CAmount FLUID_MAX_REWARD_FOR_MINING = 1000 * COIN; // Max mining block reward using fluid OP_REWARD_MINING
     static const CAmount FLUID_MAX_FOR_MINT = 1000000000 * COIN; // Max minting amount per fluid transaction
```

Comment:
Should we just set this std::max (int64_t)?

Reply:
We tested a 9223372036854775806 DYN.  When it tried to add to that number, it crashes so we implemented max limits.

Reply:
Makes sense... cool